docs/product/components/cards.html

```diff
@@ -35,39 +46,78 @@ <h1 class="fs-body3 fc-dark">Sample card title</h1>
 {% endhighlight %}
         <div class="stacks-preview--example">
             <div class="grid grid__allcells4 md:fd-column gs16">
-                <article class="grid--cell s-card bs-sm bs-hover">
-                    <h1 class="fs-body3 fc-dark">Small box shadow</h1>
-                    <p class="fs-caption fc-light">This adds a small box shadow.</p>
+                <article class="grid--cell s-card bs-sm">
+                    <h2 class="fs-body3 lh-sm fc-dark">Small box shadow</h2>
+                    <p class="fs-body1 fc-medium">This adds a small box shadow.</p>
                 </article>
-                <article class="grid--cell s-card bs-md bs-hover">
-                    <h1 class="fs-body3 fc-dark">Medium box shadow</h1>
-                    <p class="fs-caption fc-light">This adds a slightly larger box shadow.</p>
+                <article class="grid--cell s-card bs-md">
+                    <h2 class="fs-body3 lh-sm fc-dark">Medium box shadow</h2>
+                    <p class="fs-body1 fc-medium">This adds medium box shadow.</p>
                 </article>
-                <article class="grid--cell s-card bs-lg bs-hover">
-                    <h1 class="fs-body3 fc-dark">Large box shadow</h1>
-                    <p class="fs-caption fc-light">This is the largest box shadow.</p>
+                <article class="grid--cell s-card bs-lg">
+                    <h2 class="fs-body3 lh-sm fc-dark">Large box shadow</h2>
+                    <p class="fs-body1 fc-medium">This is the largest box shadow.</p>
                 </article>
             </div>
         </div>
     </div>
 </section>
 
+<section class="stacks-section">
+    {% header h2 | Linked Cards %}
+    <p class="stacks-copy">The <code class="stacks-code">.s-card</code> class can be applied to an <code class="stacks-code">&lt;a&gt;</code> tag for instances where a whole card should link somewhere.</p>
+    <p class="stacks-copy">A <code class="stacks-code">:hover</code> style is automatically added to all cards. For linked cards with a <code class="stacks-code">.bs-*</code> adding a .<code class="stacks-code">bs-hover</code> class will apply a hover style to the box shadow as well.</p>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<a href="#url" class="s-card">
+    <h2 class="fs-body3 lh-sm fc-dark">…</h2>
+    <p class="fs-body1 fc-medium">…</p>
+</a>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <div class="grid grid__allcells3 md:fd-column gs16">
+                <a href="#url" class="grid--cell s-card">
+                    <h2 class="fs-body3 lh-sm fc-dark">Sample card title</h2>
+                    <p class="fs-body1 fc-medium">This is a description or explanation.</p>
+                    <p class="fs-body1 s-link">Learn more</p>
+                </a>
+                <a href="#url" class="grid--cell s-card bs-hover bs-sm">
```

Comment:
This may not be a part of this PR (I'm not sure) but I wouldn't make `bs-hover` (or any `box-shadow` related classes) using solid colors without transparency. They look kinda off whenever you have different than white background color below element with such a shadow. In these examples, the regular shadow looks like have a transparency, but the one on hover does not.

Reply:
👍 I don't disagree but agree this isn't part of this PR.

Reply:
I think we can remove the `bs-hover` class / guidance here and replace with our newly-shipped `h:bs-sm` classes since those are based on our semi-transparent box-shadows.